### PR TITLE
New Navie UI works with the old RPC

### LINKED ISSUE #1737: New Navie Context UI works with older RPC versions

### PR DESCRIPTION
**Pull request by ahtrotta**
Fixes #1737 

I didn't add tests because this is an edge case that will become essentially obsolete in the near future once everyone is upgraded to the most recent UI and CLI.

### CODE REVIEW

File: packages/components/src/pages/ChatSearch.vue [hunk 1]
```diff
@@ -208,6 +208,41 @@ export default {
     },
   },
   methods: {
+    // This converts the old search context into the new context format
+    createContextResponse() {
+      if (!this.searchStatus) return;
+
+      const { codeObjects, sequenceDiagrams, codeSnippets } = this.searchStatus;
+      const result = [];
```
Comment (kgilpin) [inline packages/components/src/pages/ChatSearch.vue:216]: What if it’s in the new format?

File: packages/components/src/pages/ChatSearch.vue [hunk 2]
```diff
@@ -194,7 +194,7 @@ export default {
       return this.searchStatus ? this.searchStatus.step : undefined;
     },
     contextResponse() {
-      return this.searchStatus?.contextResponse;
+      return this.searchStatus?.contextResponse || this.createContextResponse();
```
[the remark this answers is not in the document]
Comment (kgilpin) [reply packages/components/src/pages/ChatSearch.vue:197]: Got it, thanks.

**Review by kgilpin — APPROVED** [review]
Ok I assume you tested this out with the old and new versions.
